Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,38 @@
 import React from 'react';
 import './App.css';
+import Display from './components/DisplayComponents/CalculatorDisplay.js'
+import Numbers from './components/ButtonComponents/NumberButton.js'
+import ActionButton from './components/ButtonComponents/ActionButton.js'
 
+
+const numbers = [ 
```

Comment:
We talked about this in standup. This is completely doable; but its just hard to do. I would devide these into separate array of objects or not and not loop through the array.